languages/python3-adk/template/src/Algorithm.py

```diff
@@ -0,0 +1,10 @@
+from Algorithmia import ADK
+
+# API calls will begin at the apply() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+def apply(input):
```

Comment:
Why doesn't the ADK template have a load function?

Reply:
Did we want to have the template as simple as possible, or did we want the default to have a load step? I'm down with having a load step be the default, but as users can't choose which template they want to see it is slightly more advanced than the default.